.github/workflows/ci.yaml

```diff
@@ -244,18 +255,16 @@ jobs:
       - id: fill-metadata
         name: Fill out a new HelmRelease resource
         run: |-
-          REF_DASHED=$(echo $GITHUB_REF | sed 's/\//-/g')
           DEPLOYMENT_NAME=$(echo $GITHUB_REPOSITORY | awk -F '/' '{print $2}')
           echo "::set-output name=deployment-name::$DEPLOYMENT_NAME"
-          echo "::set-output name=ref-dashed::$REF_DASHED"
 
           if [ "${{ matrix.environment }}" = "production" ]; then
             SANDBOX_ID="default"
             CHART_REF="$GITHUB_SHA"
             KUBERNETES_ENV="production"
             REPLICA_COUNT="3"
-            SHORT_TAG_REF=${REF_DASHED/refs-tags-/}
-            IMAGE_GLOB="${IMAGE_TAG/$SHORT_TAG_REF/*}"
+            SHORT_TAG_REF=${REF_ID/refs-tags-/}
+            IMAGE_GLOB="${REF_ID/$SHORT_TAG_REF/*}"
```

Comment:
This is because:
-  REF_DASHED is equal to REF_ID
-  IMAGE_TAG during release (for production) is equal to REF_ID